client/app/components/Input/__tests__/InputTag.spec.jsx

```diff
@@ -22,90 +29,107 @@ describe('InputTag', () => {
     expect(listbox).toBeInTheDocument();
   });
 
-  it('toggles menu options correctly', () => {
-    render(component);
+  it('toggles menu options correctly', async () => {
+    const { user } = setup(component);
     const input = screen.getByRole('textbox');
     expect(screen.queryByRole('option')).not.toBeInTheDocument();
     // toggle options by focusing on input
-    userEvent.click(input);
+    await user.click(input);
     expect(screen.getAllByRole('option')).toHaveLength(2);
   });
 
-  it('un-check existing value', () => {
+  it('un-check existing value', async () => {
     const [{ label: labelOne }] = checkboxes;
-    render(component);
+    const { user } = setup(component);
 
     const queryOptions = { name: labelOne };
     // toggle off the target option's checkbox, so we can test re-selecting it below
     let checkbox = screen.getByRole('checkbox', queryOptions);
     expect(checkbox).toBeInTheDocument();
-    userEvent.click(checkbox);
+    await user.click(checkbox);
     // after toggling it, the checkbox should disappear
     checkbox = screen.queryByRole('checkbox', queryOptions);
     expect(checkbox).not.toBeInTheDocument();
   });
 
-  it('type in value and select it', () => {
+  it('type in value and select it', async () => {
     const [{ label: labelOne }] = checkboxes;
-    render(component);
+    const { user } = setup(component);
 
     const queryOptions = { name: labelOne };
     let checkbox = screen.getByRole('checkbox', queryOptions);
     // un-check selected value so we can re-select it
-    userEvent.click(checkbox);
+    await user.click(checkbox);
     // at this point the checkbox for the option should not be visible
     expect(checkbox).not.toBeInTheDocument();
 
     // simulate searching for an option
     const input = screen.getByRole('textbox');
-    userEvent.type(input, 'One');
+    await user.type(input, 'One');
     // after selecting the option, its associated checkbox should exist
     const option = screen.getByRole('option', queryOptions);
-    userEvent.click(option);
+    await user.click(option);
 
     checkbox = screen.getByRole('checkbox', queryOptions);
     expect(checkbox).toBeInTheDocument();
   });
 
-  it('selects a value with keydown without specifying text', () => {
+  /**
+   * Skipping this one for now. react-autosuggest still checks for event.keyCode, which
```

Comment:
Have you tried using `fireEvent` instead in these skipped tests? I know it's not ideal since we want to simulate a complete interaction via `userEvent`.

Reply:
I _did_ try that - turns out it _also_ writes the `keyCode` is `0` so we end up with the same issue 😭

Reply:
Thanks for trying all this! Hmm, I'm okay with skipping this test for now, but we should have a fast follow to fix this. Since `react-autosuggest` is deprecated, I think we'll need to migrate to another library. Could you create an issue for this? Are you able to take this on?

I think we can just keep the original contents of this test since it's being skipped anyways.